_data/sites.json

```
@@ -1496,6 +1496,17 @@
         ]
     },
 
+    {
+        "name": "BrutX",
+        "url": "https://help.brutx.com/hc/fr/articles/360020090299-Sécurité-confidentialité-des-données",
```

```suggestion
        "url": "https://help.brutx.com/hc/fr/articles/360020090299",
```
Seems like the link with the diacritics is messing up our ping script. This suggestion seems to work just fine to take to the same page, so let's see how it fares.